package.json

```diff
@@ -13,6 +13,9 @@
     "url": "https://github.com/dudleycarr/ratelimit.js.git"
   },
   "author": "Dudley Carr",
+  "contributors": [
```

Comment:
[According to the npm docs](https://docs.npmjs.com/files/package.json) the `author` property is intended to be a single person and the `contributors` property is intended to be a list of people. There is no `authors` property mentioned in that doc.